templates/single.twig

```diff
@@ -72,7 +72,7 @@
 			<div class="row">
 				<div class="col-lg-12">
 					<div class="post-details">
-						{{ post.content|e('wp_kses_post')|raw }}
```

Comment:
??? why are you removing the escaping? Since we pass it raw, we should not remove the escaping